angular-spa-ui/js/admin-app/content-manager-module/contentManager-srv.js

```diff
@@ -21,9 +21,126 @@ module.exports = function () {
         });
         return tabs;
     };
+    
+    var setVisibility = function (defaultParams, params, config) {
 
+        if (!_.isEqual(defaultParams, params)) {
+            config.visibility = true;
+        }
+
+        return config;
+    }
+    
+    var findValueId = function (val, array) {
```

Comment:
indexOf() (_.indexOf):
var a = ['aaa', 'bbb', 'ccc'];
console.log(a.indexOf('bbb'));
>>  1

We use AngularJS 1.4.7. It requires IE9+. We do not caring about IE8...